evap/results/templates/result_bar.html

```diff
@@ -8,9 +8,13 @@
     {% endif %}
     {% if result.total_count %}
         data-toggle="tooltip" data-placement="left" title="{% if participants_warning %}{% trans 'Only a few participants answered this question.' %}</br></br>{% endif %}
-        {% for answer, count in result.counts.items %}
-            {{ result.question|get_answer_name:answer }}: {{ count }}/{{ result.total_count }} ({{ count|percentage_one_decimal:result.total_count }})
-            {% if not forloop.last %}</br>{% endif %}
+        {% for count in result.counts %}
+            {% with answer_name=result.question|get_answer_name:forloop.counter %}
+                {% if answer_name %}
+                    {{ answer_name }}: {{ count }}/{{ result.total_count }} ({{ count|percentage_one_decimal:result.total_count }})
+                    {% if not forloop.last %}</br>{% endif %}
+                {% endif %}
+            {% endwith %}
```

Comment:
bummer. i guess if we said that yesno questions used 1 and 2 instead of 1 and 5, that would have wider implications and complicate stuff in other places, right?

Reply:
still, overall i think the tuples are a good thing.

Reply:
i think it's not really important whether the options are 1 and 5 or anything else.

Reply:
what i'm doing here is skipping all entries where no answer name exists, otherwise we would have 5 lines in the yes/no question tooltips, where the middle three are "None: 0/n (0%)" or something like that.

Reply:
i know, and since that works around the zeroes in the counts tuple for yes/no questions, the question was whether it would make more stuff easier than it makes stuff harder if the counts tuple for yes/no questions contained only two elements instead of five so we don't need such workarounds.